proto/osmosis/protorev/v1beta1/genesis.proto

```diff
@@ -11,6 +11,4 @@ option go_package = "github.com/osmosis-labs/osmosis/v14/x/protorev/types";
 message GenesisState {
   // Module Parameters
   Params params = 1 [ (gogoproto.nullable) = false ];
-  // Hot routes that are configured on genesis
-  repeated TokenPairArbRoutes token_pairs = 2 [ (gogoproto.nullable) = false ];
```

Comment:
deprecating the use of hot routes in genesis since we will not be using it.

Reply:
Hrm, I feel like it should be added back (in a subsequent non-release affecting PR), so that ExportGenesis -> ImportGenesis still works. (So state export based testnets for instance can use this)

The ideal is that for `state' = import(export(state))`, that `state'` is effectively the same as `state`.